### DISCUSSION TITLE
Language when sorting properties

### DISCUSSION THREAD

**Discussion by tlgb** [eslint/eslint#14175] [Q&A]
Hello, please, what defines language for handling sorting properties?
I have this example of code, and I can see eslint is showing warning for props should be sorted alphabetically.
![image](https://user-images.githubusercontent.com/57944948/109966007-056da880-7cf0-11eb-938d-faab6d0859fb.png)
After vscode "source.fixAll" It shows this
![image](https://user-images.githubusercontent.com/57944948/109966324-6b5a3000-7cf0-11eb-9afb-d9f33260325f.png)
Even in node It looks correct.
![image](https://user-images.githubusercontent.com/57944948/109967025-35697b80-7cf1-11eb-9a13-dd2cb2f2e889.png)
I have tried swapping system language from czech to english -> same.
VScode is in english.
Keyboard is in english layout.

Any suggestions how to fix this? Is this even related to eslint?

**Comment by mdjermanovic** [comment]
Hi @tlgb!

This is only indirectly related to eslint. The sorting logic is in a third-party rule [`react/jsx-sort-props`](https://github.com/yannickcr/eslint-plugin-react/blob/HEAD/docs/rules/jsx-sort-props.md).

You can open an issue in the plugin's repository:

https://github.com/yannickcr/eslint-plugin-react

**Reply by tlgb** [reply]
Thank you 🙏 ! ~~I am going to ask them~~
I solved this and there is solution (in case someone find this helpfull, maybe this should not be here)
instead of using rule in eslintrc.yml
`react/jsx-sort-props: 1` just use
`react/jsx-sort-props: [1, {}]` (this is working fine)
then I want to have keys to be first so I ended up with this:
`react/jsx-sort-props: [1, { reservedFirst: true }]`
